### PR TITLE
CORS added

### LINKED ISSUE #127: CORS error
## Expected Behavior

All the API's call should be working.

## Current Behavior and Triggers

Many features are blocked due to CORS error in API call.

## Possible Solution

I tried many soln, it did'nt work at all, i also tried stackoverflow soln but it also did'nt  worked.
Adding this link https://cors-anywhere.herokuapp.com/ before all the api links will work but it will make app too slow.
Can you suggest any other soln.

## Would you like to work on this issue?

Yes

## Screenshot of issue

Screenshot (117)

### PR DESCRIPTION
**Pull request by prathmeshkhandelwal1**
Fixes #127 

### Description

CORS link added

### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)

### Non Functional Requirement
- [x] Follows the code style of this project.
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: public/client.js [hunk 1]
```diff
@@ -24,25 +24,30 @@ $('#username').keyup(function () {
...
-          username.css({
-            color: '#32d296',
-            'border-color': '#32d296'
-          })
-        }
-      })
-      .catch(e => {
-        console.log(e)
-      })
+    const profile = 'https://cors-anywhere.herokuapp.com/https://api.aashutosh.dev/' + username.val()
```
Comment (aashutoshrathi) [inline public/client.js:27]: This is not correct API endpoint

File: public/client.js [hunk 2]
```diff
@@ -24,25 +24,30 @@ $('#username').keyup(function () {
...
+    fetch(profile, {
+      mode: 'cors',
```
Comment (aashutoshrathi) [inline public/client.js:29]: No need to add this options

File: views/index.html [hunk 3]
```diff
@@ -73,14 +73,14 @@
           <div class="uk-margin">
             <div class="uk-inline">
               <span class="uk-form-icon" uk-icon="icon: user"></span>
-              <input class="uk-input uk-border-pill uk-text-small" type="text"
+              <input style="-webkit-text-fill-color: rgba(50,210,150);" class="uk-input uk-border-pill uk-text-small" type="text"
```
Comment (aashutoshrathi) [inline views/index.html:76]: Move these to CSS files

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
Please don't push until it works locally

File: public/client.js (hunk 1 above)
Comment (prathmeshkhandelwal1) [reply public/client.js:27]: > Please don't push until it works locally

I m sorry sir, but i got confused in this api call, it was giving ERROR code 404, so i thought my profile is not there.
I will fix it ASAP

**Pull request closed without merging**